src/main/java/com/neonlab/product/apis/CreateOrderApi.java

```diff
@@ -0,0 +1,41 @@
+package com.neonlab.product.apis;
+import com.neonlab.common.dto.ApiOutput;
+import com.neonlab.common.expectations.InvalidInputException;
+import com.neonlab.common.utilities.StringUtil;
+import com.neonlab.product.dtos.OrderDto;
+import com.neonlab.product.service.OrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import java.util.Objects;
+
+
+@Service
+public class CreateOrderApi {
+
+    @Autowired
+    private OrderService orderService;
+
+
+    public ApiOutput<?> createOrder(OrderDto orderDto) {
+        try {
```

Comment:
Try catch not required as GlobalExceptionHandler handles InvalidInputException. Please check

cc- @sjha24